.github/workflows/publish-new-release.yml

```diff
@@ -11,7 +11,8 @@ jobs:
   release:
     name: Publish new release
     runs-on: ubuntu-latest
-    if: github.event.pull_request.merged == true # only merged pull requests must trigger this job
+    # only merged pull requests that begin with 'release/' must trigger this job
+    if: github.event.pull_request.merged == true && startsWith(github.event.pull_request.head.ref, 'release/')
```

Comment:
This will need to handle `hotfix/` as well otherwise the code from line 25-31 does not make any sense.

Reply:
Sorry about that! Updated the check to look for `('release/' || 'hotfix/')`